test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/AutoRenewEntitiesForReconnect.java

```diff
@@ -93,6 +89,15 @@ private HapiApiSpec autoRenewAccountGetsDeletedOnReconnectingNodeAsWell() {
 				);
 	}
 
+	private HapiApiSpec accountAutoRenewalSuiteCleanup() {
+		return defaultHapiSpec("accountAutoRenewalSuiteCleanup")
+				.given().when().then(
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(GENESIS)
+								.overridingProps(disablingAutoRenewWithDefaults())
+				);
+	}
+
```

Comment:
This spec is not enabled. Is it intended?

Reply:
missed adding it to the list of specs to run. Fixed